src/main/kotlin/no/nav/syfo/job/VarselSender.kt

```diff
@@ -112,4 +111,18 @@ class VarselSender(
             }
         }
     }
+
+    fun getAllUnsendMerVeiledningVarslerLastMonth(): List<PPlanlagtVarsel> {
```

Comment:
Forslag til immutable variant:

`fun getAllUnsentMerVeiledningVarslerLastMonth(): List<PPlanlagtVarsel> {
        val allPlannedMerVeiledningVarslerLastMonth =
            databaseAccess.fetchPlanlagtMerVeiledningVarselBySendingDateSisteManed() // in max date table!
        val sentMerVeiledningVarslerLastMonth =
            databaseAccess.fetchUtsendteVarslerSisteManed().filter { it.type == VarselType.MER_VEILEDNING.name }

        return allPlannedMerVeiledningVarslerLastMonth.filter { plannedVarsel -> sentMerVeiledningVarslerLastMonth.none { plannedVarsel.fnr == it.fnr } }
    }`